scripts/core/editor3/components/Editor3.jsx

```diff
@@ -0,0 +1,224 @@
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Editor, EditorState, RichUtils, ContentState} from 'draft-js';
+
+export class Editor3 extends React.Component {
+    constructor(props) {
+        super(props);
+        var initialContentState = ContentState.createFromText(props.value);
+        this.state = {editorState: EditorState.createWithContent(initialContentState)};
+
+        this.readOnly = props.readOnly || false;
+        this.showToolbar = props.showToolbar || false;
+        this.editorFormat = props.editorFormat || [];
+        this.parentOnChange = props.onChange;
+
+        this.focus = () => this.refs.editor.focus();
+        this.onChange = (editorState) => this._onChange(editorState);
+
+        this.handleKeyCommand = (command) => this._handleKeyCommand(command);
+        this.onTab = (e) => this._onTab(e);
+        this.toggleBlockType = (type) => this._toggleBlockType(type);
+        this.toggleInlineStyle = (style) => this._toggleInlineStyle(style);
```

Comment:
Hi @ioanpocol 

I'm not sure I'm convinced by this. It makes no sense. This is the same as writing `this.toggleInlineStyle = this._toggleInlineStyle;`. And if so, why would we do this? Why not just declare `_toggleInlineStyle` directly as `toggleInlineStyle`.

In JS, to my knowledge, methods starting with underscore are an attempt at private methods that should not be tested, which is fine, but in this case, the public method does nothing on top of the private one and this is just extra code which is unnecessary and confusing.

Where did you see this example? Having seen it as an example somewhere doesn't necessarily make it good, we need to know why we are using what we are using, and if there are good arguments, then sure.

Also, I don't quite understand why we would declare methods like this when we have the ES6 syntax available.

I'm ok with keeping these changes if there is a valid explanation, but just because we've seen it somewhere else, doesn't mean it's good. Is this an example from Facebook? If so, please share it.